### PR TITLE
yesterday

### PR DESCRIPTION
**Pull request by maorfr**
all my troubles seemed so far away

use this function to template tickets that relate to cost-savings, directly into the Due Date field in jira.

:wink:

### CODE REVIEW

**Review by patjlm — APPROVED** [review]
:laughing:

File: reconcile/utils/jinja2/utils.py [hunk 1]
```diff
@@ -212,6 +213,7 @@ def process_jinja2_template(
         "url": url_makes_sense,
         "s3": lookup_s3_object,
         "flatten_dict": flatten,
+        "yesterday": lambda: (datetime.now() - datetime.timedelta(1)).strftime('%Y-%m-%d'),
```
Comment (patjlm) [inline reconcile/utils/jinja2/utils.py:216]:
```suggestion
        "yesterday": lambda: (datetime.datetime.now() - datetime.timedelta(1)).strftime('%Y-%m-%d'),
```

Comment (patjlm) [inline reconcile/utils/jinja2/utils.py:216]: or even fancier:
```suggestion
        "date_in_n_days": lambda n, format='%Y-%m-%d': (datetime.datetime.now() + datetime.timedelta(n)).strftime(format),
```
which works with negative numbers as well :wink:

Comment (maorfr) [reply reconcile/utils/jinja2/utils.py:216]: no it has to be yesterday

**Review by rporres — APPROVED** [review]

**Pull request merged**